lib/main_io.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class MainIO
+  def initialize(stdin:, stdout:)
```

Comment:
Maybe we can name these attributes/parameters something more generic than `stdin` and `stdout`? I get that you pass it `stdin` and `stdout` for the console game, but you could imagine someone wanting to use this class to extend the game to handle other kinds of input and output too.